source/times.js

```diff
@@ -26,17 +25,18 @@ import _curry2 from './internal/_curry2.js';
  */
 var times = _curry2(function times(fn, n) {
   var len = Number(n);
-  var idx = 0;
   var list;
 
   if (len < 0 || isNaN(len)) {
     throw new RangeError('n must be a non-negative number');
   }
-  list = new Array(len);
-  while (idx < len) {
-    list[idx] = fn(idx);
-    idx += 1;
+
```

Comment:
Except more readable, what's the benefit for changing `while` loop to `for` loop.

According to a [historical reason](https://github.com/ramda/ramda/pull/2744), there are almost no `for` loops in ramda  repository.